recipe/build.sh

```diff
@@ -1,13 +1,8 @@
 #!/bin/bash
 set -ex
 
-pushd HiggsAnalysis/CombinedLimit
-make CONDA=1 -j${CPU_COUNT}
-cp -rv build/bin ${PREFIX}
-cp -v build/lib/*${SHLIB_EXT} ${PREFIX}/lib
-cp -v build/lib/*.pcm ${PREFIX}/lib
-cp -v build/lib/*.rootmap ${PREFIX}/lib
-mkdir -p ${PREFIX}/include/HiggsAnalysis/CombinedLimit
-cp -rv interface ${PREFIX}/include/HiggsAnalysis/CombinedLimit/
-cp -rv build/lib/python/HiggsAnalysis $(python3 -c "import sysconfig; print(sysconfig.get_path('platlib'))")/
-popd
+python3 -c "import sysconfig; print(sysconfig.get_path('platlib'))"
+cmake ${CMAKE_ARGS} -DCMAKE_INSTALL_PREFIX=$PREFIX -DCMAKE_PREFIX_PATH=$PREFIX -S HiggsAnalysis/CombinedLimit -B build
```

Comment:
```suggestion
cmake ${CMAKE_ARGS} -DCMAKE_INSTALL_PREFIX=$PREFIX -DCMAKE_PREFIX_PATH=$PREFIX -DPYTHON_ROOT_DIR=${PREFIX} -S HiggsAnalysis/CombinedLimit -B build
```

I forgot about the `PYTHON_ROOT_DIR` hint you can give to CMake.

Reply:
Ah, seems that we were very aligned in our timing, as you _just_ committed this as I commented. :+1: